packages/hawtio/src/plugins/shared/chart/Chart.tsx

```diff
@@ -20,9 +29,10 @@ import { HawtioEmptyCard } from '../HawtioEmptyCard'
 import { HawtioLoadingCard } from '../HawtioLoadingCard'
 import { attributeService } from '../attributes/attribute-service'
 import { WatchableAttributesForm } from './WatchableAttributesForm'
+import { TableComposable, Tbody, Td, Tr } from '@patternfly/react-table'
 
 type MBeanChartData = {
-  [name: string]: MBeanChartDataEntriesTime[]
+  [name: string]: { data?: MBeanChartDataEntriesTime[]; min: number }
```

Comment:
why do we need to make `data` optional (`?`)?  we can possibly always initialise it with `[]`.